app.json

```diff
@@ -18,6 +13,10 @@
     "WHEELMAP_TOKEN": {
```

Comment:
What all is this list supposed to include? Since Wheelmap is in here I'm wondering if this enumerates tokens that are required for certain services and if Symfony/Sensiolabs Insight should be included?

Reply:
The only thing I know the placeholder entries are needed for is review apps. The env vars mentioned here get copied from the parent app (`shields-staging`) when a review app is deployed. In that sense they're only needed if you're self-hosting review apps.

Once you have a Heroku app you can always configure more settings for it.

Reply:
👍